packages/cmf/src/bootstrap.js

```diff
@@ -27,7 +28,11 @@ export function bootstrapRegistry(options) {
 	assertTypeOf(options, 'components', 'object');
 	assertTypeOf(options, 'expressions', 'object');
 	assertTypeOf(options, 'actionCreators', 'object');
+	assertTypeOf(options, 'registry', 'object');
 	registerInternals();
+	if (options.registry) {
```

Comment:
Why do you add a `registry` option ?
It's another alternative for `components/expressions/actionCreators` :/ . Let's narrow the options instead of duplicate them

Reply:
Its not an alternative. The registry is the only way to register somethings outside of the notions of `components/expressions/actionCreators/sagas`.
It lets the application use DI for specific needs.